swiftgen-cli/ArgumentsUtils.swift

```diff
@@ -47,12 +47,20 @@ enum OutputDestination: ArgumentConvertible {
     }
   }
 
-  func write(content: String) {
+  func writeIfChanged(content: String) {
```

Comment:
I think this helper function is not really needed. Directly call `write(…, onlyIfChanged: true)` everywhere we want this behavior is sufficient, having a dedicating function just for that is probably not worth it :wink: